examples/CMakeLists.txt

```diff
@@ -1,68 +1,50 @@
-cmake_minimum_required (VERSION 2.6)
-PROJECT (OPENNS_EXAMPLES)
-
-#-----------------------------------------------------------------------------
-# Setup include Directories
-#-----------------------------------------------------------------------------
-include_directories(${OPENNS_SOURCE_DIR}/api ${HDF5_INCLUDE_DIR} ${LIBXML_INCLUDE_DIR})
-
-IF(MSVC)
-
-    #ignore some warnings generated by windows.
-    add_definitions(/D_CRT_SECURE_NO_WARNINGS /D _CRT_NONSTDC_NO_DEPRECATE)
-    #definition required by HDF5 lib
-    add_definitions(-D H5_BUILT_AS_DYNAMIC_LIB)    
-    
-	#if we are building a shared library, then we need to define this symbol.
-	IF(BUILD_SHARED_LIBS)
-		add_definitions(/D BAG_DLL)
-	ENDIF(BUILD_SHARED_LIBS)
-	
-	#if we are building against a shared HDF5, then we need to define this symbol.
-	IF(BUILD_WITH_SHARED_HDF5)
-		add_definitions(/D _HDF5USEDLL_)
-		add_definitions(-D H5_BUILT_AS_DYNAMIC_LIB)
-	ENDIF(BUILD_WITH_SHARED_HDF5)
-
-ENDIF(MSVC)
+cmake_minimum_required(VERSION 3.15)
+project(OPENNS_Examples LANGUAGES C CXX)
```

Comment:
What is the actual effect of this statement? Isn't enough the project call in the parent CMakeLists?

Reply:
Addressed in c68df3591ced719c8e68f0adc97590c0c691b6e9.